scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -172,7 +175,7 @@ task ValidateFilterSetName {
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
 
   command <<<
-    set -e
+    set -ex
```

Comment:
This was failing with no error output. -x shows the bash that was run and allowed me to diagnose the issue.

Reply:
We should probably standardize on [set](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html)ting some of these options in our command blocks by default (`errexit`, `xtrace`, `pipefail`, possibly others).